### ISSUE TITLE
Someone please explain encumbrances

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#7831]
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctdisposition#restricted_usage came to exist via https://github.com/ArctosDB/arctos/issues/7605. During the latest encumbrance committee meeting (https://docs.google.com/document/d/1mWMGVXooUp-0nJKuC_lmHBBvzUgRPOyz14LGVu9pDCs/edit#heading=h.31drnsreahrc) it became clear that I have completely failed at explaining https://arctos.database.museum/info/ctDocumentation.cfm?table=ctencumbrance_action#restrict_usage, which exists to do PRECISELY what @falco-rk needs to do, and what the new "curatorial" disposition (a concept documented in https://handbook.arctosdb.org/documentation/parts.html#disposition, and another significant step away from "research quality" in my view) cannot do.

Someone please help fix this mess before it finds a way to spread.

**Comment by Jegelewicz** [comment]
> completely failed at explaining https://arctos.database.museum/info/ctDocumentation.cfm?table=ctencumbrance_action#restrict_usage, which exists to do PRECISELY what @falco-rk needs to do

I don't think it does. Encumbrances are cryptic to anyone without access to edit the data and the restrict usage encumbrance requires extra checking at the time of evaluating a loan request, where the disposition is readily apparent all over the place.

In my view, encumbrances are meant to be functional ways of redacting data. Restrict usage is the only encumbrance that doesn't do that and it doesn't seem to belong in encumbrances. Perhaps that is a misguided opinion, but it is my opinion. Also, maybe we need a definition for encumbrances in Arctos, because restrict usage is about encumbering PHYSICAL things and the others are about encumbering data.

**Comment by dustymc** [comment]
>don't think it does

That is not what Rosie said when we talked, maybe text is the barrier, I'll add another label.

> Encumbrances are cryptic

Maybe, that's why I'm asking for help.

>disposition is readily apparent

UI should ***never*** drive data.

> encumbrances are meant to be functional ways of redacting data

This does not align with the history of encumbrances. It all started with restrict usage.

>maybe we need a definition

Clearly!

**Comment by falco-rk** [comment]
Cool, glad to have caused such havoc!

**Comment by dustymc** [comment]
Adding to AWG list in the hopes that someone can at least get this pointed in a productive direction.

**Comment by dustymc** [comment]
AWG discussion: Focus on functionality would largely address this.  (Agreed, but https://github.com/ArctosDB/arctos/issues/7605 was functionality-focused and recommendations were simply not accepted??)

**Comment by dustymc** [comment]
I think this is all resolved by https://github.com/ArctosDB/arctos/issues/8241#issuecomment-2463123806

* the disposition (which had been used three times, none even kinda remotely in line with the definition...) has been removed
* restrict usage encumbrances will (tentatively) be transferred to permits (one place for "what you can do with stuff" - woohoo!)

tentatively closing
